### PR TITLE
Initial draft of roles and responsibilities. (supersedes #101)

### PR DESCRIPTION
**Pull request by hg**
new pr in place of #101, because the branch there lives on @juliepagano's fork, which would require another PR there, but she's currently busy!

### CODE REVIEW

File: docs/POLICY_ROLES_RESPONSIBILITIES.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    - Issues: view, create, comment, edit, close.
+    - Pull requests: create, comment, merge, close.
+    - Moderation of issues, pull requests, etc.
+- Has priority for participation in working group meetings (because there are limited slots available in google hangouts).
+
+### Contributors
+Contributors are people who contribute to the inclusivity working group, but do not have specific responsibilities and commitments.
+
+#### Becoming a contributor
+Anyone can be a contributor. Review the [Code of Conduct](https://github.com/nodejs/inclusivity/blob/master/CODE_OF_CONDUCT.md) before participating to familiarize yourself with expected behavior when contributing to the working group.
```
Comment (hg) [inline docs/POLICY_ROLES_RESPONSIBILITIES.md]: also, i think that this sounds misleading, as _about anyone_ can become a member as well

Comment (nebrius) [inline docs/POLICY_ROLES_RESPONSIBILITIES.md]: The wording has always been a bit tricky with this one, IMO, especially around how someone becomes a contributor. So how about this, what if we removed the `#### Becoming a contributor``` part and left this as a single paragraph.

I think the intention of the wording is to say "there is no application process", so maybe we could change it to "Anyone can be a contributor, and there is no admissions process for becoming a contributor."

Comment (hg) [inline docs/POLICY_ROLES_RESPONSIBILITIES.md]: yeah, that sounds better

File: docs/POLICY_ROLES_RESPONSIBILITIES.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    - Pull requests: create, comment, merge, close.
+    - Moderation of issues, pull requests, etc.
+- Has priority for participation in working group meetings (because there are limited slots available in google hangouts).
+
+### Contributors
+Contributors are people who contribute to the inclusivity working group, but do not have specific responsibilities and commitments.
+
+Anyone can be a contributor, and there is no admissions process for becoming a contributor. Review the [Code of Conduct](https://github.com/nodejs/inclusivity/blob/master/CODE_OF_CONDUCT.md) before participating to familiarize yourself with expected behavior when contributing to the working group.
+
+### Responsibilities
```
Comment (nebrius) [inline docs/POLICY_ROLES_RESPONSIBILITIES.md]: Just noticed this, can you add an extra `#`? There should be four here.

**Comment by nebrius** [comment]
:shipit: LGTM, thanks @juliepagano and @hg! I'll leave this open for a bit to give other people a chance to comment before merging later this afternoon (assuming no comments)

**Comment by nebrius** [comment]
Merging (woo!)

**Pull request merged**